src/project.rs

```diff
@@ -73,6 +73,10 @@ pub struct Project {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub serve_address: Option<IpAddr>,
 
+    /// The mode to use when mapping scripts onto Roblox
```

Comment:
```suggestion
    /// The mode to use when mapping scripts into Roblox.
    /// Can be either `Class` or `RunContext` and determines whether script
    /// behavior is set using the `RunContext` property or the script's `ClassName`.
```

Something like this is probably better, though I am by no means a documentation expert.

Reply:
Will fix everywhere, thanks